### PR TITLE
FF8: allow external textures from magic.fs

### PR DESCRIPTION
**Pull request by myst6re**
## Summary

Related to #29 
Some textures was not correctly detected.

### ACKs

- [X] I have updated the [Changelog.md](https://github.com/julianxhokaxhiu/FFNx/blob/master/Changelog.md) file
- [ ] I did test my code on FF7
- [X] I did test my code on FF8

### CODE REVIEW

**Comment by julianxhokaxhiu** [comment]
Can you please fix the conflict with the master branch? Thank you

**Comment by myst6re** [comment]
> Can you please fix the conflict with the master branch? Thank you

Done and ready!

File: Changelog.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ## FF8
 
+- Graphics: add external textures replacement for magic.fs textures ( https://github.com/julianxhokaxhiu/FFNx/pull/646 )
```
Comment (julianxhokaxhiu) [inline Changelog.md:7]:
```suggestion
- Graphics: Add external textures replacement for magic.fs textures ( https://github.com/julianxhokaxhiu/FFNx/pull/646 )
```

**Review by julianxhokaxhiu — APPROVED** [review]
Spotted a minor optional fix on the Changelog but LGTM. Let me know if you want to fix it or if i can continue like it is :) Thanks a lot for this awesome PR!

**Comment by myst6re** [comment]
> Spotted a minor optional fix on the Changelog but LGTM. Let me know if you want to fix it or if i can continue like it is :) Thanks a lot for this awesome PR!

No problem, fixed!

**Review by julianxhokaxhiu — APPROVED** [review]

**Pull request merged**